tests/sqlparser_snowflake.rs

```diff
@@ -2709,6 +2709,20 @@ fn parse_use() {
                 Ident::with_quote(quote, "my_schema")
             ])))
         );
+        std::assert_eq!(
+            snowflake().verified_stmt(&format!("USE ROLE {0}my_role{0}", quote)),
+            Statement::Use(Use::Role(ObjectName(vec![Ident::with_quote(
+                quote,
+                "my_role".to_string(),
+            )])))
+        );
+        std::assert_eq!(
```

Comment:
```suggestion
        assert_eq!(
            snowflake().verified_stmt(&format!("USE ROLE {0}my_role{0}", quote)),
            Statement::Use(Use::Role(ObjectName(vec![Ident::with_quote(
                quote,
                "my_role".to_string(),
            )])))
        );
        assert_eq!(
```
if these were autoimported?

Reply:
Fixed across the board